src/Mono.Android/Mono.Android.targets

```diff
@@ -128,6 +129,7 @@
         SourceFiles="$(IntermediateOutputPath)__dex\classes.dex"
         DestinationFiles="$(OutputPath)mono.android.dex"
     />
+    <Touch Files="$(OutputPath)mono.android.dex" />
```

Comment:
Either is fine, but the more important question here is: what is triggering this target? Could you check the diagnostic build output and see which targets depend on these _GenerateMonoAndroidDex* targets?